code.js

```diff
@@ -1,3 +1,41 @@
-are_isomorphic(graph1, graph2) {
-    return undefined;
+function are_isomorphic(graph1, graph2) {
+    if (graph1.length != graph2.length){return false}
+    v1 = Vnum(graph1)
+    v2 = Vnum(graph2)
+    if (v1 != v2){return false}
+    if (same(graph1, graph2)){return true}
+    for (let i=0; i < graph1.length; i++){
+        if (isomorphic(graph1, graph2)){return true}
+    }
+    return false
+}
+function isomorphic(graph1, graph2){
```

Comment:
As far as I can see, this is not correct. You would have to check all permutations, not just move the last node to the beginning.